lib/virtfs/ext3/base.rb

```diff
@@ -0,0 +1,50 @@
+module VirtFS::Ext3
+  class FS
+    attr_accessor :mount_point, :superblock, :root_dir
+
+    attr_accessor :entry_cache, :dir_cache
+
+    DEF_CACHE_SIZE = 50
+
+    def self.match?(blk_device)
+      begin
+        blk_device.seek(0, IO::SEEK_SET)
+        VirtFS::Ext3::Superblock.new(blk_device)
+        return true
+      rescue => err
+        return false
+      end
+    end
+
+    def initialize(blk_device)
+      blk_device.seek(0, IO::SEEK_SET)
+      @superblock  = Superblock.new(blk_device)
```

Comment:
Just noticed that you invoke "Superblock.new" here but on line 12 above the call is "VirtFS::Ext3::Superblock.new".
If this line works then the more explicit line is probably not necessary, right?